tutorial-week-4.md

```diff
@@ -1,8 +1,29 @@
-##Tutorial: Package.json
+##Tutorial: Adding a Package.json file
 
-- `cd` folder, correct branch
-- `npm init` (gives you package.json)
-- install development dependencies
-- in `scripts` key add: "start"- browser-sync, "sass"- node-sass
+###Step 1:
```

Comment:
Before you get into the meat of the tutorial, be sure to explain what the purpose of a `package.json` is: what problem is this a solution to?